cognite/neat/graph/loaders/_rdf2dms.py

```diff
@@ -276,18 +274,22 @@ def _upload_to_cdf(
                 skip_on_version_conflict=True,
             )
         except CogniteAPIError as e:
+            result = UploadResult[InstanceId](name="Instances", issues=read_issues)
             result.error_messages.append(str(e))
             result.failed_upserted.update(item.as_id() for item in e.failed + e.unknown)
             result.created.update(item.as_id() for item in e.successful)
+            yield result
         else:
-            for instance in itertools.chain(upserted.nodes, upserted.edges):
-                if instance.was_modified and instance.created_time == instance.last_updated_time:
-                    result.created.add(instance.as_id())
-                elif instance.was_modified:
-                    result.changed.add(instance.as_id())
-                else:
-                    result.unchanged.add(instance.as_id())
-        return result
+            for instance_type, instances in {"Nodes": upserted.nodes, "Edges": upserted.edges}.items():
+                result = UploadResult[InstanceId](name=instance_type, issues=read_issues)
+                for instance in instances:  # type: ignore[attr-defined]
+                    if instance.was_modified and instance.created_time == instance.last_updated_time:
+                        result.created.add(instance.as_id())
+                    elif instance.was_modified:
+                        result.changed.add(instance.as_id())
+                    else:
+                        result.unchanged.add(instance.as_id())
+                yield result
```

Comment:
This change to the code is to get a better feedback to the user.

Instead of 
![image](https://github.com/cognitedata/neat/assets/60234212/56aa1f66-514f-4d63-bb85-e17995953a39)

We now get 
![image](https://github.com/cognitedata/neat/assets/60234212/1d94965d-b9da-409e-9fbc-350609a57145)